Map component

HugoDelgado18 opened the pull request.

Dan-Y-Ko commented:
Alright, I checked out your branch locally instead of just looking at changes in the repo. Some thoughts:

1. What is the reason for changing the version for react-native-gesture-handler?
2. I recommend not using hardcoded dimensions for the map. 
3. I can't seem to drag the markers at all.
4. The markers aren't stacking on top of each other, it's just that the map is zoomed way out. If you zoom in, you can see they are working fine. May need to look into setting initial zoom level but idk if that's within the scope of this ticket.
5. Not related to the ticket but I think it would be better if you keep commit messages short and to the point and note your summary and / or issues you ran into in the description for your pr.

HugoDelgado18 commented:
> Alright, I checked out your branch locally instead of just looking at changes in the repo (was busy till now). Some thoughts:
> 
> 1. What is the reason for changing the version for react-native-gesture-handler?
> 2. I recommend not using hardcoded dimensions for the map.
> 3. I can't seem to drag the markers at all.
> 4. The markers aren't stacking on top of each other, it's just that the map is zoomed way out. If you zoom in, you can see they are working fine. May need to look into setting initial zoom level but idk if that's within the scope of this ticket.
> 5. Not related to the ticket but I think it would be better if you keep commit messages short and to the point and note your summary and / or issues you ran into in the description for your pr.

Hello, Dan wasn't sure who you were on DIscord to reply to directly.
1. I'll ahead and change that.
2. I fixed the Markers issue already, They used to be stacked because I was using ParseInt to convert the coordinates but that was rounding the coordinates making them all stacked. I switched it to a parseFloat.
3. To drag you need to press and hold until the marker then you can move. You should be able to. Can you message me on discord so I can see it now working? Hugo18 
4. I agree I was worried I wasn't doing it right so I asked robin for advice on how to make a proper commit message yesterday my most recent commit is "HOT-11 added MapWidget with functionality to MapList" 
Robin told me not to worry about the description but I'll add one next time.

Dan-Y-Ko commented on src/components/MapWidget.tsx:
```diff
@@ -0,0 +1,67 @@
+
+const styles = StyleSheet.create({
+  mapContainer: {
+    backgroundColor: '#fff',
+    alignItems: 'center',
+    justifyContent: 'center',
```
I just noticed this. I don't think these styling are necessary.

HugoDelgado18 replied:
I just made the changes. I believe the Redux from dev will be the next Pull so I will hold off on pushing so I can make any changes if conflicts are present.
